pkgs/common-updater/unstable-updater.nix

```
@@ -25,7 +27,9 @@ let
 
     # Get info about HEAD from a shallow git clone
     tmpdir="$(${coreutils}/bin/mktemp -d)"
-    ${git}/bin/git clone --bare --depth=1 "$url" "$tmpdir"
+    ${git}/bin/git clone --bare --depth=1 \
+      ${lib.optionalString (branch != null) " --branch " + branch} \
```

I think it looks more natural like this (the space before the flag is unnecessary since we have it outside the ${}

```suggestion
      ${lib.optionalString (branch != null) "--branch ${branch}"} \
```

---

done, PR updated.